tests/full.test.ts

```diff
@@ -513,6 +512,25 @@ for (let index = 0; index < storages.length; index++) {
       }
     });
 
+    it('test delete leaf near middle node', async () => {
+      const sto = getTreeStorage();
+      const mt = new Merkletree(sto, true, 10);
+
+      const keys = [7n, 1n, 5n];
```

Comment:
1. What about testing the same but with test data which creates a kind of symmetric tree structure (branches grow right but not left in the SMT)? 
2. Do we have any tests which deletes a leaf which HAS ANOTHER LEAF as a sibling and as a result the simbling should be "bubbled up" to more than one level?